include/deal.II/base/geometric_utilities.h

```diff
@@ -45,16 +45,30 @@ namespace GeometricUtilities
      * Return spherical coordinates of a Cartesian point @p point.
      * The returned array is filled with radius, azimuth angle $\in [0,2 \pi)$
      * and polar/inclination angle $ \in [0,\pi]$ (ommited in 2D).
+     *
+     * In 3D the transformation is given by
+     * @f{align*}
+     *  r &= \sqrt{x^2+y^2+z^2} \\
+     *  \theta &= {\rm atan}(y/x) \\
+     *  \phi &= {\rm acos} (z/r)
+     * @f}
```

Comment:
Can you submit the changes in this file separately? They are independent and uncontroversial.